src/ergw_config.erl

```diff
@@ -21,6 +21,7 @@
 
 -define(DefaultOptions, [{plmn_id, {<<"001">>, <<"01">>}},
 			 {accept_new, true},
+			 {gtp_idle_timer_secs, 21600}, %% def value = 6 hours
```

Comment:
that should IMHO be a per APN config option (maybe with a global default)
@mgumz comments?

Reply:
per APN seems OK, even we usually have only one c-node container running per APN. but as long multi APNs are supported they might have a different idle_timer. are there any options to receive such idle-timer value from control plane protocols? At least RADIUS knows a idle_timeout Attribute:

https://freeradius.org/rfc/rfc2865.html#Idle-Timeout

Reply:
Checked the 3GPP 29.274 (GTPv2-C protocol), unlike Radius, could not see any indicator flags in any of the messages of this nature.  Section 7.2.9.2 in 29.274, implies that Application should manage "PDN connection inactivity timer"  and trigger a Delete Bearer Request upon it expiry (as in the ergw). (optionally "Cause" IE indicating expiry of this timer can be sent) . 
Will add the per APN idle timer config in the ergw.

Reply:
There is no idle timeout in GTP. but RADIUS and DIAMETER NASREQ both have `Idle-Timeout`. Those attributes can certainly be used to overwrite a configure timeout. Minimum wins obviously.

Don't confuse that with the somewhat related, but different Continuous Time Period (CTP) or Discrete Time Period (DTP) charging, see TS 29.244 and TS 32.299 (Sect. 6.5.7).